src/Framework/Config/FileIo.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gacela\Framework\Config;
+
+final class FileIo implements FileIoInterface
+{
+    public function existsFile(string $filePath): bool
+    {
+        return file_exists($filePath);
+    }
+
+    /**
+     * @return mixed
+     */
+    public function include(string $filePath)
```

Comment:
I think maybe it would make more sense to split this method into a new interface?

I don't see a real "relationship" between the `file_exists` and the `include` 🤔